siriuspy/siriuspy/pwrsupply/csdev.py

```diff
@@ -895,6 +912,9 @@ def _get_ps_common_propty_database():
             'type': 'float', 'value': 1.0, 'prec': 4, 'unit': 's',
             'lolo': 1e-2, 'low': 1e-2, 'lolim': 1e-2,
             'hilim': 4096, 'high': 4096, 'hihi': 4096},
+        # Period Sync Pulse
+        'PeriodSyncPulseCount-Mon': {
```

Comment:
the name `PeriodSyncPulseCount` (ans its corresponding PS variable) is not intuitive for what it meas. Talk @gabrielbrunheira to think a better name.